### PR TITLE
EVG-3367: run legacy runners in amboy jobs

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: units/legacy_alerts.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+			message.Fields{
+				"runner":  "alerts",
+				"message": "alerts are disabled, exiting",
+				"id":      j.ID(),
+			})
+		return
+	}
+
+	if len(config.SuperUsers) == 0 {
+		grip.Warning(message.Fields{
```
Comment (richardsamuels) [inline units/legacy_alerts.go]: Nit, but i'd call this an alert

File: units/legacy_monitoring.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+func makeLegacyMonitorRunnerJob() *legacyMonitorJob {
+	return &legacyMonitorJob{
+		Base: job.Base{
+			JobType: amboy.JobType{
+				Name:    legacyMonitorRunnerJobName,
+				Version: 0,
+			},
+		},
+	}
```
Comment (richardsamuels) [inline units/legacy_monitoring.go]: SetDependency

File: units/legacy_alerts.go [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+func NewLegacyAlertsRunnerJob(env evergreen.Environment, id string) amboy.Job {
+	j := makeLegacyMonitorRunnerJob()
+	j.env = env
+	j.SetID(fmt.Sprintf("%s.%s", legacyAlertRunnerJobName, id))
+	return j
+}
+
+func (j *legacyAlertsJob) Run(ctx context.Context) {
+	if j.env == nil {
```
Comment (richardsamuels) [inline units/legacy_alerts.go]: MarkComplete

File: units/legacy_alerts.go [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+		grip.Error(message.WrapError(err, message.Fields{
+			"runner":    "alerts",
+			"status":    "failed",
+			"id":        j.ID(),
+			"operation": "finding super users",
+		}))
+
+		j.AddError(errors.Wrap(err, "problem getting superuser list"))
+		return
+	}
```
Comment (richardsamuels) [inline units/legacy_alerts.go]: What happens if superUsers is nil here? i.e., if all of the super users that were listed are invalid

File: units/legacy_alerts.go (hunk 3 above)
Comment (john-m-liu) [reply units/legacy_alerts.go]: I think jobs are auto marked complete once Run returns?

File: units/legacy_alerts.go (hunk 4 above)
Comment (tychoish) [reply units/legacy_alerts.go]: we would have logged a bunch at reasonably high levels. Nothing is really different here than it was in the old model.

**Pull request merged**